src/startContentProcessor.js

```diff
@@ -1,41 +1,46 @@
-﻿// Dependent on DOM structure and javascript of workflowy
-// exportIt is a custom workflowy function so can only be called in the page context
-
-
-var opml = extractOpmlData();
-var parsedOpml = $.parseXML(opml);
-var xml = $(parsedOpml);
-
-var siteTitle = $('.project.selected > .name').text();
-
-var converter = new Converter(xml, siteTitle);
-converter.GetZippedHtmlFiles(function (data) {
-  saveAs(data, "Prototype.zip");
+﻿define([
+  'lib/jquery-2.1.4.min',
```

Comment:
Up to you, but some people prefer to use require `path` config to map `jquery` to the current jquery version, rather than having the exact filename with version number in here (and same goes for other libraries). Means you don't have to make a change to every file in your codebase if you upgrade your version. It's an easy change anyway, but it's a bit messy.